setup.py

```diff
@@ -22,12 +22,11 @@
         'Environment :: Web Environment',
         'Framework :: Django',
         'Framework :: Django :: 2.2',
-        'Framework :: Django :: 3.1',
```

Comment:
Is there a reason for us to drop 3.1 support, but keep 2.2?

Reply:
Django 2.2 is an LTS release and supported until April.  Django 3.1 EOLed back in December.

Source: https://www.djangoproject.com/download/#supported-versions